EventGenerator/src/CRYEventGenerator_module.cc

```diff
@@ -72,6 +76,20 @@ namespace mu2e {
     mf::LogInfo("CRYEventGenerator") << oss.str();
   }
 
+  void CryEventGenerator::endSubRun(art::SubRun &subrun)
+  {
+    // All inputs (except getLiveTime) in CosmicLivetime are not used in the livetime calculation
+    // Livetime is calculated internally by cry: cryGen->getLiveTime()
+    std::unique_ptr<CosmicLivetime> livetime(new CosmicLivetime(1,
+                                                                cryGen->getSubboxLength(),
+                                                                cryGen->getMinShowerEn()/1000.,
+                                                                cryGen->getMaxShowerEn()/1000.,
+                                                                1.8e4,
```

Comment:
How is this constant calculated?  is it merely a conversion factor?  Or does it have physical units that are somehow constant for this generator?  Either way, could you please add a comment?